python/ray/experimental/serve/examples/echo_error.py

```diff
@@ -0,0 +1,37 @@
+"""
+Example of error handling mechanism in ray serve
+
+- HTTP server should respond with "internal error"
+- ray.get(handle.remote(33)) should raise RayTaskError
```

Comment:
Took me a bit to understand the point of these two comments. Please start with writing that this actor class always returns an exception, which is handled in different ways depending on how you call it